api/enki.py

```diff
@@ -784,9 +803,31 @@ def incremental_import(self, since):
         return new_titles, circulation_changes
 
     def update_circulation(self, since):
-        """Process circulation events that happened since `since`."""
+        """Process circulation events that happened since `since`.
```

Comment:
The behavior of this method hasn't changed -- we still process all the events between `since` and the current time. But we now may be making multiple API calls.